index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+   <head>
+      <meta charset="utf-8">
+      <title>Adopt-a-Dog homepage: Look here to learn more about Seattle's premiere dog adoption agency</title>
+      <link rel="stylesheet" type="text/css" media="only screen and (min-width: 481px)" href="stylesheets/desktop/stylesheet-index.css">
+      <link rel="stylesheet" type="text/css" media="only screen and (max-width: 480px)" href="stylesheets/mobile/stylesheet-index-mobile.css">
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+   </head>
+   <body>
+      <header>
+        <div id="header-checkout-all">
+         <div class="header-checkout">
+            <img class="cart-logo" src="images/cart-icon.svg" alt="Shopping cart icon">
+            <p id="header-price">$1234.56</p>
+          </div>
+            <a class="button" href="checkout.html">Checkout</a>
+            <div class="header-intro">
+            <img class="header-logo" src="images/adoptadog-logo.svg" alt="Adopt-a-dog logo">
+            <h2 class="header-title"><em>Seattle's premiere dog adoption agency</em></h2>
+         </div>
+       </div>
+      </header>
+      <nav class="global-navigation">
+         <a href="index.html" class="nav-link">Home</a>
+         <p> / </p>
+         <a href="dogs.html" class="nav-link-2">Dogs</a>
+         <p> / </p>
+         <a href="blog.html" class="nav-link-2">Blog</a>
+      </nav>
+      <section class="index-content">
+         <div class="index-1">
```

Comment:
This div and index-2 are not resizing properly, they are being set by their content. Instead you should assign them some %width and then have their content resize to fill them. As is the dog image is not changing size and index-1 stays the same size while the text shrinks and has less size at smaller resolutions.